### PR TITLE
Fixed the newly renamed README.md files

### PR DESCRIPTION
**Pull request by unixb0y**
Especially regarding line breaks and markdown tables

### CODE REVIEW

**Comment by unixb0y** [comment]
I fixed the issues and my latest commit includes the sign-off.
Can someone help with the merge?

**Comment by mcmasterg** [comment]
There is sign off on the latest, but not the previous commits

**Comment by unixb0y** [comment]
@mcmasterg I know, I thought it is not necessary as those commits were all on my working branch.
What to do now?

**Comment by mcmasterg** [comment]
A few options:
-Amend your existing commits and add the DCO phrase. You can do this by using "git rebase -i 6411a75^1) and select "r" to reword commits. Then it will prompt you for each commit where you will paste the DCO to the end of the message
-Alternatively, squash those readme commits into one (using the same command, but use s instead of r). Then it should be pretty easy to add the DCO rewording to the few remaining commits

If this is still unclear, maybe we can chat on g-chat, irc, or something like that?

Also can you clarify the newline thing: you made the more consistent? Were some of them using Windows newlines?

**Comment by unixb0y** [comment]
Thanks, I'll merge them into one, then! Haven't used Git (except the regular commit, push flow for my private projects) in a while now... 😄 
IRC would be cool in general!
Well, .txt files show a newline by just typing it. In markdown, it is different: You have to add 2 spaces to the end of the line.
I fixed an open issue earlier today which stated to rename all README.txt files to README.md but they still had to be "converted" regarding that newline thing and regarding tables.

**Comment by mcmasterg** [comment]
Cool. If I'm online I idle in freenode ##openfpga under nick digshadow

**Comment by unixb0y** [comment]
Rebase done, DCO still says "sign-off is missing".
Should I close the pull request and open a new one?

**Comment by JohnDMcMaster** [comment]
Hmm I only see two files changed. These were the conlficting ones right? Did you do a "commit -m" instead of a "commit -am"?

File: fuzzers/015-clbnffmux/README.md [hunk 1]
```diff
@@ -1,42 +1,39 @@
...
-CLB.SLICE_X1.BFFMUX.B3 31_24
-CLB.SLICE_X1.CFFMUX.B0 31_35
-CLB.SLICE_X1.CFFMUX.B1 31_38
-CLB.SLICE_X1.CFFMUX.B2 31_37
-CLB.SLICE_X1.CFFMUX.B3 31_36
-CLB.SLICE_X1.DFFMUX.B0 30_58
-CLB.SLICE_X1.DFFMUX.B1 31_61
-CLB.SLICE_X1.DFFMUX.B2 31_62
-CLB.SLICE_X1.DFFMUX.B3 31_60
+Purpose:
```
Comment (mithro) [inline fuzzers/015-clbnffmux/README.md]: Would you mind formatting them this way?

<pre lang="no-highlight"><code>
# CLBnFFMUX Fuzzer

## Purpose

Document nFFMUX family of CLB muxes  

## Algorithm

## Outcome

```
CLB.SLICE_X0.AFFMUX.B0 30_00
CLB.SLICE_X0.AFFMUX.B0 30_00
CLB.SLICE_X0.AFFMUX.B0 30_00
... more lines ...
```

From manual O6 testing ;

```
X0 AOUTMUX O6  
X0 BOUTMUX O6  
```

</code></pre>

which will then render like this;

> # CLBnFFMUX Fuzzer
> ## Purpose
> 
> Document nFFMUX family of CLB muxes  
> 
> ## Algorithm
> 
> ## Outcome
> 
> ```
> CLB.SLICE_X0.AFFMUX.B0 30_00
> CLB.SLICE_X0.AFFMUX.B0 30_00
> CLB.SLICE_X0.AFFMUX.B0 30_00
> ... more lines ...
> ```
> 
> From manual O6 testing ;
> 
> ```
> X0 AOUTMUX O6  
> X0 BOUTMUX O6  
> ```

**Comment by unixb0y** [comment]
@mithro Hi Tim, you’re welcome. I’m happy to help!
Once I have more time (1 month from now approx) and get a little introduction on _how_ I would like to contribute with the bitstream documentation as well.
Cheers, Dave 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: fuzzers/015-clbnffmux/README.md (hunk 1 above)
Comment (unixb0y) [reply fuzzers/015-clbnffmux/README.md]: Sure thing!

Comment (unixb0y) [reply fuzzers/015-clbnffmux/README.md]: Done !

**Comment by mithro** [comment]
@unixb0y The changes look good!

**Comment by mithro** [comment]
@unixb0y I'm going to merge this pull request now!

FYI - To do lists in markdown you use;
<pre lang="no-highlight"><code> * Item 1
 * Item 2
</code></pre>

Which renders like this;
> * Item 1
> * Item 2

You can also use dash;
<pre lang="no-highlight"><code> - Item 1
 - Item 2
</code></pre>

Which renders like this;
> - Item 1
> - Item 2

The important things is that the line starts with a space and then a star then another space. For ordered lists you can do;
<pre lang="no-highlight"><code> 1. Item 1
 2. Item 2
</code></pre>

Which renders like this;
> 1. Item 1
> 2. Item 2

**Pull request merged**